ospfd/ospf_vty.c

```diff
@@ -3543,6 +3543,31 @@ static void show_ip_ospf_interface_sub(struct vty *vty, struct ospf *ospf,
 				vty_out(vty,
 					"  No backup designated router on this network\n");
 		} else {
+			nbr = ospf_nbr_lookup_by_addr(oi->nbrs, &DR(oi));
+			if (nbr) {
+				if (use_json) {
+					json_object_string_add(
+						json_interface_sub, "DRId",
```

Comment:
please use camelCase in JSON keys, as per our development guidelines, unless there is a good reason not to

Reply:
Changed to camelCase.